.github/workflows/ci.yml

```diff
@@ -30,3 +30,32 @@ jobs:
           tsc --noEmit
           npm run lint
           npm test
+  build-node-version-16-18:
```

Comment:
Given that this step now does more than it did before, how about renaming it? 

> Build node 16 and run storybook
Or such.